projects/Mallard/src/helpers/files.ts

```diff
@@ -35,19 +35,29 @@ export const fileIsIssue = (file: File): file is IssueFile =>
 export const ensureDirExists = (dir: string): Promise<void> =>
     RNFS.mkdir(dir).catch(() => Promise.resolve())
 
+const extractEditionIds = (editionList: EditionInterface[]) =>
+    editionList.map(e => e.edition)
+
 /*
 We always try to prep the file system before accessing issuesDir
 */
 export const prepFileSystem = async (): Promise<void> => {
     await ensureDirExists(FSPaths.issuesDir)
     await ensureDirExists(FSPaths.downloadRoot)
+    const editionsList = await editionsListCache.get()
```

Comment:
The cache list gets updated fairly often so it should be fine to rely on this. @jimhunty may have other ideas?

Reply:
Yea it should be fine I think. `prepFileSystem` runs before all download behaviour and therefore should have a decent idea of what editions there are. There may be edge cases though